projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -17,12 +17,87 @@
 
 <ng-container *ngIf="record">
   <header class="row">
-    <h1>{{ record.metadata.title }}</h1>
+
+    <!-- TITLE -->
+    <ng-container *ngIf="record.metadata.title">
+      <ng-container *ngFor="let title of record.metadata.title">
+        <ng-container *ngIf="title.type === 'bf:Title'">
+          <h1>{{ title._text }}</h1>
+        </ng-container>
+      </ng-container>
+    </ng-container>
   </header>
   <section>
     <dl class="row mb-0">
+
+      <!-- ALTERNATIVE GRAPHICS -->
+      <ng-container *ngIf="record.metadata.ui_title_altgr">
+        <ng-container *ngFor="let altgr_title of record.metadata.ui_title_altgr; first as isFirst">
+          <ng-container *ngIf="isFirst; else elseBlock">
+            <dt class="col-sm-4 offset-sm-2 offset-md-0 label-title" translate>
+              Title
+            </dt>
+            <dd class="col-sm-7 col-md-8 mb-0">
+              {{ altgr_title | titlecase }}
+            </dd>
+          </ng-container>
+          <ng-template #elseBlock>
+            <dt class="col-sm-4 offset-sm-2 offset-md-0">
+              &nbsp;
+            </dt>
+            <dd class="col-sm-7 col-md-8 mb-0">
+              {{ altgr_title | titlecase }}
+            </dd>
+          </ng-template>
+        </ng-container>
+      </ng-container>
+
+      <!-- VARIANT TITLE -->
+      <ng-container *ngIf="record.metadata.ui_title_variants">
+        <ng-container *ngFor="let variant_title of record.metadata.ui_title_variants; first as isFirst">
+          <ng-container *ngIf="isFirst; else elseBlock">
+            <dt class="col-sm-4 offset-sm-2 offset-md-0 label-title" translate>
+              Variant title
+            </dt>
+            <dd class="col-sm-7 col-md-8 mb-0">
+              {{ variant_title | titlecase }}
+            </dd>
+          </ng-container>
+          <ng-template #elseBlock>
+            <dt class="col-sm-4 offset-sm-2 offset-md-0">
+              &nbsp;
+            </dt>
+            <dd class="col-sm-7 col-md-8 mb-0">
+              {{ variant_title | titlecase }}
+            </dd>
+          </ng-template>
+        </ng-container>
+      </ng-container>
+
+      <!-- RESPONSIBILITY -->
+      <ng-container *ngIf="record.metadata.ui_responsibilities">
+        <ng-container *ngFor="let responsibility of record.metadata.ui_responsibilities; first as isFirst">
+          <ng-container *ngIf="isFirst; else elseBlock">
+            <dt class="col-sm-4 offset-sm-2 offset-md-0 label-title" translate>
+              Statement of responsibility
+            </dt>
+            <dd class="col-sm-7 col-md-8 mb-0">
+              {{ responsibility | titlecase }}
+            </dd>
```

Comment:
'col-sm-4' + 'offset-sm-2' + 'col-sm-7' = 13 ? bootstrap grid layout limit is 12 isn't it ?
Same comment for any metadata container for this view